conda_build/metadata.py

```diff
@@ -292,10 +293,12 @@ def evalidate_model():
     model.nodes.extend(["Call", "Attribute", "Tuple", "List", "Dict"])
     model.allowed_functions += [
         "int",
+        "float",
         "str",
         "list",
         "dict",
         "tuple",
+        "len",
     ]
     model.attributes += [
```

Comment:
Do you think we'll need `join` and `replace` in this list too?

Reply:
Yeah good idea.